openfe/setup/atom_mapping/perses_mapper.py

```diff
@@ -43,7 +67,7 @@ def __init__(self, allow_ring_breaking: bool = True,
             if mappings must strictly preserve chirality, default: True
         use_positions: bool, optional
             this option defines, if the
-        coordinate_tolerance: float, optional
+        coordinate_tolerance: openmm.unit.Quantity, optional
```

Comment:
A note, we say "optional" but none of the type hints say Optional[...].. maybe that's something we should also remember to fix.